src/design/fallback-images/fallback-images.stories.mdx

```diff
@@ -0,0 +1,21 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+// Importing a `.svg` directly, results in a failed build and importing
+// the `.svg.twig` file causes linting errors (since they are not tracked).
+// Therefore, we must track a twig file to satisfy CI.
+import fallbackImage from '../demo/fallback_images.twig';
+
+<Meta title="Design/Fallback Images" />
+
+# Fallback Images
+
+We changed the way we displayed articles, but it requires all articles to have some sort of image. That can kind of disturb the flow of things when there isn't a background image present.
+
+To combat this, we've created a few different fallback images. The goal was to not have them compete too much with intentional images, but we also wanted them to be on brand. This design also leaves room to add more categories if desired.
+
+The captions below each fallback image aren't set in stone, moreso just ideas that these designs could be used for.
```

Comment:
Can we make this more about the _current_ solution and less about the change from the past, which will become less and less relevant over time?